Sources/PackageLoading/Module+PkgConfig.swift

```diff
@@ -129,6 +131,25 @@ extension SystemPackageProvider {
     }
 }
 
+/// Removes discardable flags such as:
+/// cFlags: -fabi-version, -fno-omit-frame-pointer
+/// libs:
+func removeDiscardableFlags(cFlags: [String], libs: [String]) -> (cFlags: [String], libs: [String]) {
+    let cFlags: [String] = cFlags.flatMap { flag in
+        switch flag {
+        case "-fabi-version":
```

Comment:
We need to take care here since compiler allows different formats: `-fabi-version=1`, `-fabi-version 1`
In first case we need to check the prefix and discard that flag.
In second case we need to discard flag as well as the next argument.